data/database.py

```diff
@@ -2152,13 +2155,25 @@ def upstream_registry_hostname(self) -> str:
 
 class RedHatSubscriptions(BaseModel):
     """
-    Represents internal Red Hat subscriptions for customers
+    Represents store for users' RH account numbers
     """
 
-    user_id = ForeignKeyField(User, backref="subscription")
+    user_id = ForeignKeyField(User, backref="account_number")
     account_number = IntegerField(unique=True)
 
 
+class OrganizationRhSkus(BaseModel):
+    """
+    Represents sku to org binding for
+    RH subscriptions
+    """
+
+    id = IntegerField(primary_key=True)
+    subscription_id = IntegerField(unique=True)
+    user_id = IntegerField()
```

Comment:
What's the user_id for if org_id already exists?

Reply:
The user_id is so we can track who assigned the sku to the org. That way when a user leaves an org we can remove all bindings in that table matching the user_id and org_id.

Reply:
In this case wouldn't we want that field to be a FK referencing the user table?

Reply:
Would it be ok to have multiple FK constraints on the user table since org_id already does that?

Reply:
I don't see why not. That way, you don't have to enforce that constraint in the application.